message.go

```diff
@@ -2,7 +2,12 @@ package subee
 
 // Message is an interface of the subscribed message.
 type Message interface {
+	Acknowledger
+	Data() []byte
+}
+
+// Acknowledger is an interface to send ACK/NACK.
+type Acknowledger interface {
```

Comment:
Because it is necessary to add ack / nack (), so added `Acknowledger `.
https://github.com/wantedly/subee/pull/5/files#diff-2392601cebbdfe90fdac28a6618bd8cbR8